lib/assessment.rb

```diff
@@ -13,28 +13,28 @@ def longest(enum)
   # enumerable supplied (i.e. apply the `+` operator)
   # Assume each entry implements a `+` method
   def sum(enum)
-    # Your code goes here
+    sum+=num
```

Comment:
No, sorry. When you want to sum across the elements in an array (or similar), which tool do you think is the one to use?